satellite/satellitedb/overlaycache.go

```diff
@@ -466,8 +466,11 @@ func (cache *overlaycache) UpdateOperator(ctx context.Context, nodeID storj.Node
 	}
 
 	updated, err := convertDBNode(updatedDBNode)
+	if err != nil {
+		return nil, Error.Wrap(err)
```

Comment:
The transaction must be rolled back in this case. I pushed a fix.

Reply:
I don't think I'm seeing the fix?

Reply:
This is because a later commit actually removed the transaction in this method. So there is no need of commit and rollback anymore.

Reply:
ah okay!